x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_process_list.tsx

```diff
@@ -160,9 +184,15 @@ export const ProcessListWithCounts = memo(function ProcessListWithCounts({
     ];
   }, []);
 
+  const children = useSelector(hasMoreChildren);
+  const ancestors = useSelector(hasMoreAncestors);
+
   return (
     <>
       <StyledBreadcrumbs breadcrumbs={crumbs} />
+      {children === true || ancestors === true ? (

```

Comment:
you could either: `const showWarning = children === true || ancestors === true` then `showWarning && <StyledLineageLimitWarning .... />` or just place the boolean in the parenthetical and remove the null option